documents/motions/06.2022.1 - Regional Organizations.md

```diff
@@ -47,12 +48,14 @@ The WCA shall acknowledge Regional Organizations as Members to support the Objec
          3. Official website.
          4. Official logo.
          5. A copy of its current bylaws and, if applicable, regulations, which are in compliance with the Bylaws and Regulations of the WCA.
+            1. Applicants must provide an English translation of their Bylaws as well as their original Bylaws in the relevant local language.
          6. A list of its principal directors and officers and their titles.
          7. A clear description of the area that is covered by the Regional Organization.
          8. A report on past and current WCA related activities.
 5. Acknowledgement of Regional Organizations
-   1. The WCA Board shall have the right to acknowledge the Regional Organization, based on a voting in the WCA Board of Directors with at least supermajority.
+   1. The WCA Board shall have the right to acknowledge the Regional Organization.
```

Comment:
Same question as [here](https://github.com/Epride/wca-documents/pull/34#discussion_r1623207826) regarding the use of "shall" instead of "must".